root/report/VideoRelationshipsOnNonVideos.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import RecordingList from './components/RecordingList.js';
+import ReportLayout from './components/ReportLayout.js';
+import type {ReportDataT, ReportRecordingT} from './types.js';
+
+component VideoRelationshipsOnNonVideos(...{
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportRecordingT>) {
+  return (
+    <ReportLayout
+      canBeFiltered={canBeFiltered}
+      description={l(
+        `This report shows recordings not marked as video, but that use
+         relationships meant only for video recordings. Either they should be
+         marked as video, or the relationships should be moved to a related
+         video recording.`,
```

Comment:
Same as in the other pull request, it would be helpful to know the reported relationship types:
```suggestion
        `This report shows recordings not marked as video, but that use
         relationships meant only for video recordings (“artwork”,
         “choreographer”, “cinematographer”, “design”, “graphic design”,
         “illustration”, “video appearance”, “video director”). Either they
         should be marked as video, or the relationships should be moved to a
         related video recording.`,
```

After taking a long time at copying/pasting relationship type names from the comments in code, maybe it would be better to make a comma list of those? Translators would be less likely to make divergent translations, and if a relationship type is renamed, we wouldn’t have to remind of renaming it here as well.

Reply:
A comma list sounds like a good idea, so I did that. The only issue with the implementation is that we want quotes, but the quotes are not currently translatable. I wonder if we should add a `QuotedCommaList`... this should be good enough for now though.